m3u8/parser.py

```diff
@@ -280,15 +284,20 @@ def _parse_cueout(line, state):
         state['current_cue_out_duration'] = res.group(1)
         state['current_cue_out_scte35'] = res.group(2)
 
-
 def _parse_cueout_start(line, state, prevline):
     param, value = line.split(':', 1)
     state['current_cue_out_duration'] = value
+    # Try Elemental flavor first
```

Comment:
How about to use two functions to parse this value:

```python
def _cueout_elemental(prevline):
    return ...


def _cueout_envivio(prevline):
    return ...


# and then

state['x'] = _cueout_elemental(prev) or _cueout_envivio(prev)
```

What do you think? My intention was to make it clear.